Header header

# position in encoder ticks
# velocities in encoder ticks per 0.5 sec.
int32 position_right
int32 position_left
float32 velocity_right
float32 velocity_left
